internal/subscription/delay.go

```diff
@@ -0,0 +1,201 @@
+package subscription
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/razorpay/metro/internal/topic"
+)
+
+// subs-delay-30-seconds, subs-delay-60-seconds ... subs-delay-600-seconds
+const delayTopicNameFormat = "%v-delay-%v-seconds"
+
+const (
+	defaultMinimumBackoffInSeconds    uint = 10
+	lowerBoundMinimumBackoffInSeconds uint = 0
+	upperBoundMinimumBackoffInSeconds uint = 600
+
+	defaultMaximumBackoffInSeconds    uint = 600
+	lowerBoundMaximumBackoffInSeconds uint = 0
+	upperBoundMaximumBackoffInSeconds uint = 600
+
+	defaultMaxDeliveryAttempts    int32 = 5
+	lowerBoundMaxDeliveryAttempts int32 = 5
+	upperBoundMaxDeliveryAttempts int32 = 100
+)
+
+var (
+	// ErrInvalidMinBackoff ...
+	ErrInvalidMinBackoff = errors.New("min backoff should be between 0 and 600 seconds")
```

Comment:
Instead of hardcoding "0" "600" in the error string, use the constants `lowerBoundMinimumBackoffInSeconds` `upperBoundMinimumBackoffInSeconds` in the error string? This way, in case the ranges are changed in future, error string need not be explicitly changed.

Reply:
Done.